### PR TITLE
Update GitpodWorkspaceTooManyRegularNotActive severity level

### PR DESCRIPTION
**Pull request by princerachit**
## Description

Update the severity of the alert to critical and change the threshold based on previous alerts generated by `GitpodWorkspaceStuckOnStopping` which is usually alerted right after the `GitpodWorkspaceTooManyRegularNotActive` threshold breach.

## Related Issue(s)

Fixes https://github.com/gitpod-io/gitpod/issues/8051

## How to test

NA

## Release Notes

```release-note
None
```

### CODE REVIEW

**Comment by sagor999** [comment]
/hold
hold until this https://github.com/gitpod-io/runbooks/pull/322 will get merged first.

**Review by sagor999 — CHANGES_REQUESTED** [review]

File: operations/observability/mixins/workspace/rules/components/workspaces/alerts.libsonnet [hunk 1]
```diff
@@ -61,16 +61,16 @@
...
+            'for': '15m',
             annotations: {
-              runbook_url: 'none',
+              runbook_url: 'https://github.com/gitpod-io/runbooks/blob/main/runbooks/GitpodWorkspaceTooManyRegularNotActive.md',
               summary: 'too many running but inactive workspaces',
               description: 'too many running but inactive workspaces',
             },
             expr: |||
-              gitpod_workspace_regular_not_active_percentage > 0.10 AND sum(gitpod_ws_manager_workspace_activity_total) > 100
+              gitpod_workspace_regular_not_active_percentage > 0.15 AND sum(gitpod_ws_manager_workspace_activity_total) > 100
```
[the remark this answers is not in the document]
Comment (princerachit) [reply operations/observability/mixins/workspace/rules/components/workspaces/alerts.libsonnet:73]: Given that we are not doing any other filtering for other alerts including `GitpodWorkspaceStuckOnStopping` and previously this alert also did not have such filter, I will prefer to just follow the pattern. I looked at previous stats in grafana and the number seems reasonable.

[the remark this answers is not in the document]
Comment (sagor999) [reply operations/observability/mixins/workspace/rules/components/workspaces/alerts.libsonnet:73]: ah, I see. Hm. That might backfire in the future when we have more clusters though. But I guess we cam deal with that we can get there.

**Review by sagor999 — APPROVED** [review]

**Comment by kylos101** [comment]
/unhold
runbook merged

**Pull request merged**